modules/ossm-threescale-webassembly-module-mapping-rule-object.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-webassembly-module-mapping-rule-object_{context}"]
+= The 3scale WebAssembly module `mapping_rule` object
+
+The `mapping_rule` object is part of an array in the `mapping_rules` object.
+
+The `mapping_rule` object fields specify:
```

Comment:
s/specify:/specify the following:/

or maybe:

s/specify:/specify the following information:/